resources/test/fixtures/flake8_django/DJ13.py

```diff
@@ -0,0 +1,17 @@
+from django.db.models.signals import pre_save
```

Comment:
It'd be preferable, I think, to copy over the existing test fixtures directly (https://github.com/rocioar/flake8-django/blob/master/tests/fixtures/model_dunder_str.py). That way, we can test for 1:1 compatibility with the existing plugin.